src/Lykke.Service.PayPushNotifications/Controllers/NotificationController.cs

```diff
@@ -0,0 +1,44 @@
+﻿using AutoMapper;
+using Lykke.Common.Api.Contract.Responses;
+using Lykke.Service.PayPushNotifications.Core.Services;
+using Lykke.Service.PayPushNotifications.Filters;
+using Microsoft.AspNetCore.Mvc;
+using Swashbuckle.AspNetCore.SwaggerGen;
+using System.Collections.Generic;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace Lykke.Service.PayPushNotifications.Controllers
+{
+    [ValidateActionParametersFilter]
+    [Route("api/[controller]/[action]")]
+    public class NotificationController: Controller
```

Comment:
Just a tip: in 2.1 you can inherit from ControlleBase and use [ApiController] attribute to have automatic validation.